src/app/design/guidelines/call-out-info/index.html

```diff
@@ -50,10 +50,7 @@
   </stache-page-anchor>
 
   <p>
-    To communicate a message that a background process triggers, use a toast. Background processes include processes that users initiate and system processes that users need to be aware of.
+    To communicate a message that a background process triggers, use a <a routerLink="/components/toast">toast</a>. Background processes include processes that users initiate and system processes that users need to be aware of. Since toasts contain information the user must see, the user must manually dismiss the toast - the toast will not automatically close.
   </p>
-  <p>
-    Depending on the nature of the message, you can specify whether the toast fades automatically or requires users to dismiss it. In general, <stache-code>success</stache-code> and <stache-code>info</stache-code> toasts should fade automatically, while <stache-code>warning</stache-code> and <stache-code>danger</stache-code> toasts should require users to manually dismiss the messages since they often require follow-up actions.
```

Comment:
Are we changing the guidance on which types of toasts should fade? Just curious since I don't see this added back anywhere else in the PR.

Reply:
I believe we're doing away with auto-dismissing of any kind on toasts and popovers.

Reply:
Good deal. Just wanted to make sure. Thanks!